examples/datasets/load_limo_dataset.py

```diff
@@ -0,0 +1,64 @@
+"""
+========================================================
+Import single subject from LIMO data set into MNE-Python
+========================================================
+
+Here we a define function to extract the eeg signal data
+from the LIMO structures in the LIMO dataset, see [1]_ and:
+
+    https://datashare.is.ed.ac.uk/handle/10283/2189?show=full
+
+    https://github.com/LIMO-EEG-Toolbox
+
+The code allows to:
+
+Fetch single subjects epochs data for the LIMO data set.
+Epochs information (i.e., sampling rate, number of epochs per condition,
+number and name of EEG channels per subject, etc.) is extracted from
+the LIMO .mat files stored on disk.
+If files are not found, the function mne.datasets.limo.load_data() will
+automatically download the data from a remote repository.
+
+mne.datasets.limo.load_data() creates a custom info and
+epochs structure in MNE-Python.
+Missing channels can be interpolated if desired.
+
+.. note:: Downloading the LIMO dataset for the first time can take some time (8 GB).
+
+
+References
+----------
+.. [1] Guillaume, Rousselet. (2016). LIMO EEG Dataset, [dataset].
+       University of Edinburgh, Centre for Clinical Brain Sciences.
+       https://doi.org/10.7488/ds/1556.
+"""
+
+# Authors: Jose C. Garcia Alanis <alanis.jcg@gmail.com>
+#
+#
+# License: BSD (3-clause)
+
+
+# import necessary packages
+import os.path as op
+
+import scipy.io
+import numpy as np
+import mne
+
+from mne.datasets import limo
+
+print(__doc__)
+
+
+# fetch data from subject 1
+limo_epochs = limo.data_path(subject=1)
+
+# inpect epochs info
+limo_epochs.info
+
+# look at metadata
+limo_epochs.metadata
```

Comment:
Hm? If this is not printed, we don't actually see it right? Also .head is probably better.